apps/chat-e2e/src/ui/webElements/header.ts

```diff
@@ -23,4 +24,16 @@ export class Header extends BaseElement {
   public rightPanelToggle = this.getChildElementBySelector(
     HeaderSelectors.rightPanelToggle,
   );
+
+  public newEntityButton = this.getChildElementBySelector(
+    SideBarSelectors.newEntity,
```

Comment:
please, move the locator to `HeaderSelectors`

Reply:
done